am-ui/webcomponents/search-results-webcomponent/src/i18n.jsx

```diff
@@ -1,5 +1,5 @@
 import i18n from 'i18next';
```

Comment:
Translations file better to be adjusted if we are adding more values. For a general value like the no results hierarchy is needed but if we add more values specific to a category then it should be [categories.nameOfCategory.key...] so for example "categories.accomodation.month"

Reply:
Yes I agree - I decided not to create a hierarchy for this ticket since there isn't a lot of translations currently with the intention to create one when we need it (since we currently only have one category). Would you prefer me doing it now?

Reply:
There really is not much content so no problem, just to keep it in mind if more values are added, always more costly to change all that when there is more content.